webdriver/tests/bidi/storage/set_cookie/invalid.py

```diff
@@ -36,15 +36,15 @@ async def test_cookie_http_only_invalid_type(bidi_session, test_page, domain_val
         "cookie\x0Fname",
         "cookie;name",
     ])
-async def test_cookie_name_invalid_value(bidi_session, test_page, domain_value, name):
+async def test_cookie_name_invalid_value(set_cookie, test_page, domain_value, name):
```

Comment:
Firefox API doesn't actually fail to set cookies with these names

Reply:
ack. Let's keep it out of scope of this PR